boilerplates/express/files/express-entry.ts

```diff
@@ -116,6 +118,17 @@ async function startServer() {
     app.post("/_telefunc", handlerAdapter(telefuncHandler));
   }
 
+  if (BATI.has("drizzle") && !(BATI.has("telefunc") || BATI.has("trpc"))) {
+    app.use(express.json()); // Parse & make HTTP request body available at `req.body`
+    app.post("/api/todo/create", async (req, res) => {
+      const newTodo: TodoInsert = req.body;
+
+      const result = await db.insert(todoTable).values({ text: newTodo.text });
+
+      res.status(201).send({ message: "New Todo Created", result });
+    });
```

Comment:
You should move that directly into `boilerplates/drizzle`, in the same fashion as `boilerplates/shared-server/files/server/vike-handler.ts`, i.e. create a universal handler (I'll call this handler **A** for future references in the next comments).